### PR TITLE
Ports fishkeeping from Paradise

### PR DESCRIPTION
**Pull request by ghost**
# Document the changes in your pull request
Ports fishkeeping from Paradise. It can be an alternative food source for the kitchen. More information about it you can get on the official wiki of ss220: https://paradisestation.org/wiki/index.php/Fish

Also it ports some seafood, but since i am a noob, recipes of that food are ported not very good(they all are made through crafting and food processor, because i didnt find recipes code for grilling or cooking).

I open it second time because first PR with this was fucked by my low skills in using github. I hope now it will work normally.

# Wiki Documentation
Add a guide for fishkeeping.
Add a new vendomat to vendor list - CritterCare
Add more recipes to cooking guide.
Add aquariums into guide to construction.

 # Changelog
🆑Author of original PR: FalseIncarnate
Original PR: ss220-space/Paradise@9074629

rscadd: Adds a new sidejob - fishkeeping
rscadd: Adds a new vedor machine - CritterCare
rscadd: Adds some new toys
rscadd: Adds a new kit
rscadd: Adds some new food
rscadd: Adds some new cooking recipes
imageadd: Adds icons for fish items and seafood.
/🆑

### CODE REVIEW

**Review by maxion12345 — APPROVED** [review]
Line 162 in the food and drinks file your food is indented over at the top. also they really went crazy with the indentations, the comments are shoved all the way over and it seems each if statement gets an extra one. The uplink code has about four new empty lines at the bottom as well.

You've also commited changes to yarn.lock, please revert these changes. They are generated by testing in private servers, and should be discarded prior to commiting and stashing changes made. 

also apparently I can approve code lol

Can't wait to have fish fry :)

**Comment by ghost** [comment]
> Line 162 in the food and drinks file your food is indented over at the top. also they really went crazy with the indentations, the comments are shoved all the way over and it seems each if statement gets an extra one. The uplink code has about four new empty lines at the bottom as well.
> 
> also apparently I can approve code lol

New empty lines are the ghosts of my other branches, and appear here because i dont properly understand how does github work.

**Comment by maxion12345** [comment]
> > Line 162 in the food and drinks file your food is indented over at the top. also they really went crazy with the indentations, the comments are shoved all the way over and it seems each if statement gets an extra one. The uplink code has about four new empty lines at the bottom as well.
> > also apparently I can approve code lol
> 
> New empty lines are the ghosts of my other branches, and appear here because i dont properly understand how does github work.

No worries. You probably just hit enter a few times and then committed those as a change. Not a big deal, just pointing out they exist! 
As for how GitHub works? Best I can explain is that you take a copy of our code, modify it, then commit the changes and do a pull request to take your committed changes and apply them to the code. You should be able to undo or revert changes by file, so check what google has to say on the matter

**Comment by ghost** [comment]
> > > Line 162 in the food and drinks file your food is indented over at the top. also they really went crazy with the indentations, the comments are shoved all the way over and it seems each if statement gets an extra one. The uplink code has about four new empty lines at the bottom as well.
> > > also apparently I can approve code lol
> > 
> > 
> > New empty lines are the ghosts of my other branches, and appear here because i dont properly understand how does github work.
> 
> No worries. You probably just hit enter a few times and then committed those as a change. Not a big deal, just pointing out they exist! As for how GitHub works? Best I can explain is that you take a copy of our code, modify it, then commit the changes and do a pull request to take your committed changes and apply them to the code. You should be able to undo or revert changes by file, so check what google has to say on the matter

Ok

**Comment by ghost** [comment]
Why do this checks take so long

**Comment by bruhlookatthisdood** [comment]
> Why do this checks take so long

a maintainer needs to authorize before checks start.

**Comment by ghost** [comment]
Okay this PR is killed because of merge conflicts that are too complex to view, so no fishes.

**Comment by NCRLouTenant** [comment]
RIP, was probably my most anticipated PR. Maybe one day.

**Comment by JamieD1** [comment]
> Okay this PR is killed because of merge conflicts that are too complex to view, so no fishes.

What do you mean? Its a dmi and a map change..
Need help?

**Comment by ghost** [comment]
> > Okay this PR is killed because of merge conflicts that are too complex to view, so no fishes.
> 
> 
> 
> What do you mean? Its a dmi and a map change..
> 
> Need help?

Yes, i need help. The problem is that i cant resolve it through the browser, and also i can't normally switch to this branch on PC for some reason(i dont get all changed files when i do it). Also i dont know how to normally fix merge conflicts.

**Comment by ghost** [comment]
> RIP, was probably my most anticipated PR. Maybe one day.

Maybe i can just copy+paste all of this changes into a new    up-to-date branch(if anything else wouldn't help) when i will finish with my infection project. So no worries.

**Comment by ghost** [comment]
Okay lets go i will fix this shit

**Comment by ghost** [comment]
Wow, i fixed merge conflict?!

**Comment by ghost** [comment]
CHECKS PLEASE

File: code/game/objects/items/storage/fancy.dm [hunk 1]
```diff
@@ -359,3 +359,6 @@
 	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
 	STR.max_items = 8
 	STR.set_holdable(list(/obj/item/reagent_containers/food/snacks/tinychocolate))
+
+
+
```
Comment (JamieD1) [inline code/game/objects/items/storage/fancy.dm]:
```suggestion
	STR.set_holdable(list(/obj/item/reagent_containers/food/snacks/tinychocolate))
```

File: code/modules/fish/fish_eggs.dm [hunk 2]
```diff
@@ -0,0 +1,68 @@
+
```
Comment (JamieD1) [inline code/modules/fish/fish_eggs.dm]:
```suggestion
```

File: code/modules/fish/fish_items.dm [hunk 3]
```diff
@@ -0,0 +1,207 @@
+
```
Comment (JamieD1) [inline code/modules/fish/fish_items.dm]:
```suggestion
```

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 4]
```diff
@@ -2024,3 +2024,5 @@
 			wounded_part.heal_damage(0.25, 0.25)
 		M.adjustStaminaLoss(-0.25*REM) // the more wounds, the more stamina regen
 	..()
+
+
```
Comment (JamieD1) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]:
```suggestion
	..()
```

File: code/modules/vending/critercare.dm [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+					/obj/item/fish_eggs/electric_eel = 5, /obj/item/fish_eggs/shrimp = 10,)
+
+	contraband = list(/obj/item/fish_eggs/babycarp = 5)
+	refill_canister = /obj/item/vending_refill/crittercare
+	default_price = 10
+	extra_price = 50
+
+
+
+
```
Comment (JamieD1) [inline code/modules/vending/critercare.dm]:
```suggestion
	extra_price = 50
```

**Review by JamieD1 — CHANGES_REQUESTED** [review]
Please can you try and get rid of all this empty space and obviously fix the errors.

**Comment by ghost** [comment]
> Please can you try and get rid of all this empty space and obviously fix the errors.

Okay

**Comment by JamieD1** [comment]
Cornflakes

**Comment by JamieD1** [comment]
Reopen when fixed

**Comment by ghost** [comment]
How to reopen in a phone

**Pull request closed without merging**